<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerBox</class>
 <widget class="QWidget" name="ViewerBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Данные</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="metadata_layout"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Номер события</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_num_event"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_count_event"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_show_event">
       <property name="text">
        <string>Посмотреть</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_stat">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tw_viewer"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sb_num_event</tabstop>
  <tabstop>sb_count_event</tabstop>
  <tabstop>btn_show_event</tabstop>
  <tabstop>btn_stat</tabstop>
  <tabstop>tw_viewer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
